### ISSUE TITLE
Remove infinite scroll

### ISSUE THREAD

**Issue by jcn** [waffleio/waffle.io#1875]
It is very difficult to be able to figure out the relative size of the column with infinite scrolling in place. If there was a way to show the entire scrollbar for a given column while only loading the cards that are visible at any given time, that would be great.

**Comment by CodeMan99** [comment]
There is a count of cards at the top of any given column.

**Comment by jcn** [comment]
To me it's more visual than that. If I'm working on a column and looking through the cards that I have available, it's difficult when I'm in the middle of the list to know whether I'm halfway down the list, a quarter of the way, etc. The card count is interesting, but not that useful if I don't know where in the list of "30" cards I am.

**Comment by CodeMan99** [comment]
So what should be in the blank space where cards have yet to load?

* Empty Cards? This will make it really difficult to parse "pagination" load errors. May also cause drawing issues when an empty card box is loaded and need to become larger.
* Empty Space? Again, what should be displayed in the case of a loading error?

Infinite/dynamic scrolling is the industry solution to this problem.

How should you handle infinite scrolling? Plan by labeling cards as you manage a column. Cut down on the count by filtering by label.

**Comment by jcn** [comment]
It's definitely a good question, and I'm mostly basing my comments on the joy I get from using Trello. It looks like they do some some kind of dynamic loading on page load. If you load this board: https://trello.com/b/rbpEfMld/data-science and look at the second "Data News" column you'll see that immediately after page load they continue to load the rest of the cards in that list.

**Comment by CodeMan99** [comment]
:+1: for linking an example. I think the waffle team should have a good starting point if they decide to tackle this issue now.

**Comment by roll** [comment]
It's not objective but I also just dosn't like how infinite scroll feels. Why you can't preload all the cards? It's not a landing page we can wait another 100 ms and allocate another 100kb.

**Comment by joerick** [comment]
I agree with @jcn on this one. The scrollbar is a subtle but useful indicator of the size of the list. I often don’t realize how long my backlog is because the scrollbar makes it look like about 15 issues.

I’d be happy with either empty space or perhaps 50% opacity cards while they are loading, if I scroll too fast.

**Comment by niemela** [comment]
+1
